### ISSUE TITLE
Cross-post attachments to upload service like CloudApp

### ISSUE THREAD

**Issue by ZevEisenberg** [br1sk/brisk#36]
Might be a stretch, but I always upload my attachments to CloudApp (unless they contain sensitive information). It would be nice to get this automatically.

**Comment by keith** [comment]
It would be nice to have a solution for this that was generic to everyone and didn't require separate accounts 🤔 

I guess if we did this we'd also want to edit the open radar text and link to the uploaded attachments.

**Comment by ZevEisenberg** [comment]
Yep. I always upload and put it in the Notes section as "screenshot also uploaded at …"

I figured this one was a stretch goal, but wanted to file it so it's tracked somewhere.
